desktop/conf.dist/hue.ini

```diff
@@ -1029,7 +1029,12 @@
     # # options='{"url": "presto://localhost:8080/hive/default", "has_impersonation": true}'
     # ## Kerberos.
     # # options='{"url": "presto://localhost:8080/hive/default?KerberosKeytabPath=/path/to/keytab&KerberosPrincipal=principal&KerberosRemoteServiceName=service&protocol=https"'
-
+    # ## LDAPS enabled over HTTPS.
```

Comment:
Any chance to also add it to https://github.com/cloudera/hue/blob/master/desktop/conf/pseudo-distributed.ini.tmpl#L1014 ? (to keep the dev ini in sync too)

Reply:
Yes, will do that.